README.md

```diff
@@ -0,0 +1,228 @@
+orders. Security is ensured through robust authentication and authorization mechanisms utilizing JWT tokens.</p>
+
+
+<p align="center" style="font-size: 1.2em;">
+  <a href="#technologies" style="text-decoration: none;">🚀 <strong>Technologies</strong></a> •
+  <a href="#getting-started" style="text-decoration: none;">🛠️ <strong>Getting Started</strong></a> •
+  <a href="#domain-models" style="text-decoration: none;">📚 <strong>Domain Models</strong></a> •
+  <a href="#user-roles" style="text-decoration: none;">👥 <strong>User Roles</strong></a> •
+  <a href="#endpoints" style="text-decoration: none;">🔗 <strong>Endpoints</strong></a>
+  <a href="#challenges" style="text-decoration: none;">📄 <strong>Challenges</strong></a>
+</p>
+
+
+<h2 id="technologies"> Technologies</h2>
+<ul style="list-style: none">
+ <li><img src="img/java.png" alt="" width="25" style="position: relative; top: 5px;"> Java 17</li>
+ <li><img src="img/spring.png" alt="" width="25" style="position: relative; top: 5px;"> Spring framework <i><small>(boot, data, security)</small></i> </li>
+ <li><img src="img/lombok.png" alt="" width="25" style="position: relative; top: 5px;"> Lombok</li>
+ <li><img src="img/mapstruct.png" alt="" width="25" style="position: relative; top: 5px;"> MapStruct</li>
+ <li><img src="img/mysql.png" alt="" width="25" style="position: relative; top: 5px;"> MySql 8</li>
+ <li><img src="img/hibernate.png" alt="" width="25" style="position: relative; top: 5px;"> Hibernate</li>
+ <li><img src="img/liquibase.svg" alt="" width="25" style="position: relative; top: 5px;"> Liquibase</li>
+ <li><img src="img/docker.png" alt="" width="25" style="position: relative; top: 5px;"> Docker</li>
+</ul>
+
+
+<div style="display: flex; align-items: center;">
+    <img src="img/swagger.png" alt="" width="50" style="margin-right: 10px;">
+    <h1 id="start" style="margin: 0;">Swagger Documentation</h1>
+</div>
+<p>
+    Swagger is integrated into the project to provide comprehensive API documentation and facilitate testing of endpoints. With Swagger, you can visualize and interact with the API's resources without having any of the implementation logic in place.
+</p>
+<p>To access the Swagger UI and explore the API documentation:</p>
+<ol>
+    <li>Once the application is running, navigate to <a href="http://localhost:8080/swagger-ui.html">http://localhost:8080/swagger-ui.html</a> in your web browser.</li>
+    <li>You will see the Swagger UI interface, where you can view all available endpoints, their descriptions, request parameters, and response schemas.</li>
+    <li>Use the interactive features of Swagger UI to make requests directly from the browser and observe the responses.</li>
+</ol>
+
+
+<div style="display: flex; align-items: center;">
+    <img src="img/start.png" alt="" width="50" style="margin-right: 10px;">
+    <h1 id="getting-started" style="margin: 0;">Getting Started</h1>
+</div>
```

Comment:
add detailed auth section about how to register/login and how to use the received token. Also add user and admin credentials to readme